resources/views/pages/faq.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('layouts.master')
+
+@section('main_content')
+
+    @include('layouts.header', ['header' => 'FAQ'])
+
+    <div class="container -padded">
+        <div class="wrapper">
+            <div class="container__block -narrow">
+                <h3>What is Rogue?</h3>
+                <p>Rogue is an internal tool where staff and interns review and view more information about reportbacks. It is the tool you should use when looking for content for a sponsor, decks, media, etc!</p>
+
+                <h3>Is Rogue an internal or external term?</h3>
+                <p>Rogue is an internal term - it's named for <a href="https://en.wikipedia.org/wiki/Rogue_(comics)" target="_blank">this</a> superhero!</p>
+
+                <h3>There's a lot of terms flying around, can I get some definitions?</h3>
+                <ul class='list'>
+                    <li><b>Signup:</b> A signup is created when a user signs up for a campaign.</li>
+                    <li><b>Reportback:</b> This is a term that the organization has traditionally used to define whether or not a member has completed the action or not once. If a member submits proof of their action, they have reported back. If a member sends in multiple proofs of their action, this is still just one report back.</li>
```

Comment:
`traditionally used to define whether or not a member has completed the action or not once.`
I think there is either an extra word or a missing word here! "once" might be extra?

Reply:
great eyes, @katiecrane ! Removed the `once` :)